.env.example

```
@@ -11,6 +11,11 @@ JWT_REFRESH_SECRET=your-refresh-secret-key
 JWT_ACCESS_EXPIRATION=15m
 JWT_REFRESH_EXPIRATION=7d
 
+JWT_ADMIN_ACCESS_SECRET=your-admin-access-secret-key
+JWT_ADMIN_REFRESH_SECRET=your-admin-refresh-secret-key
+JWT_ADMIN_ACCESS_EXPIRATION=15m
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_ADMIN_ACCESS_EXPIRATION key should go before the JWT_ADMIN_ACCESS_SECRET key

(UnorderedKey)